extensions/stats/testdata/istio/metadata-exchange_filter.yaml

```diff
@@ -19,46 +19,6 @@ spec:
             config:
               configuration: envoy.wasm.metadata_exchange
```

Comment:
can you remove this configuration or make it empty? it's not used.

Reply:
Do we have a test verify metadata exchange filter with empty configuration? I am just worrying the wasm layer might bypass onConfigure call when it is not provided.

Reply:
I tried it without in one of the tests, it worked. However, I'm not sure it is not bypassing, so let's leave it as-is then.